README.md

```diff
@@ -1,463 +1,161 @@
-#### Confused? Have a question?
-
-There is a Matrix/FreeNode IRC channel `#LibreLingo`, instructions for joining:
 
-* Matrix: [`#LibreLingo`](https://riot.im/app/#/group/+librelingo:matrix.org) on `matrix.org`
-* IRC: connect to `irc.freenode.net` with your favourite client and join `#LibreLingo`
+## Roadmap
+- [ ] Officially releasing Basque course
+- [ ] Make it possible to contribute to courses through GitHub
+- [ ] Improving course creation UX over Github
 
-## Articles
 
-- [Why I built LibreLingo](https://dev.to/kantord/why-i-built-librelingo-280o)
-- [Practice TDD by creating a LibreLingo-based language-learning app](https://dev.to/kantord/practice-tdd-by-creating-a-librelingo-based-language-learning-app-4apf)
+## Contribution
+### Become a contributor:
+#### are you a developer?
+You can help LibreLingo by testing it and submitting feature requests or bug reports: [here](https://github.com/kantord/LibreLingo/issues/new). If you want to get in touch, you can use my contact details on [my GitHub profile](https://github.com/kantord).
+Go through the dev docs [here](https://librelingo.app/docs/).
 
-## License
-LibreLingo is licensed under the AGPL-3.0 license. In addition, course content and other creative content might be licensed under different licenses, such as CC.
+Still got questions? Our Matrix/FreeNode IRC channel is `#LibreLingo`, join the dev community there and feel free to ask anything.
+* Matrix: [`#LibreLingo`](https://riot.im/app/#/group/+librelingo:matrix.org) on `matrix.org`
+* IRC: connect to `irc.freenode.net` with your favourite client and join `#LibreLingo` 
 
-This image shows a summary of the licensing of our dependencies:
+#### are you a linguaphile?
+Learning new language is an awesome experience right? If you wanna let others experience that too, make a course on LibreLingo! Get started [here](https://github.com/kantord/LibreLingo/tree/main/docs/courses).
 
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fkantord%2FLibreLingo.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Fkantord%2FLibreLingo?ref=badge_large)
+We are working on the process of creating courses, to make it more streamlined, if you have an idea to make it better, do share with us, create an [issue](https://github.com/kantord/LibreLingo/issues/new).
+Note: You are always encouraged to release the courses under a free license.
 
-### Attributions
-#### Mascot
-
-<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><br />Mascot images are released under <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
+#### are you a wordsmith?
+Translate the LibreLingo UI(LLUI) to make it more accessiable, you can start [here](https://poeditor.com/join/project?hash=jKiKfen2Ye).
 
+### Attributions:
```

Comment:
if we're doing major changes in the readme then I think it's also worth considering to move attributions to a separate file and perhaps organize them better. also create an issue to somehow put the attributions also on the web app

Reply:
Before creating the issue I want to dissuss on it a bit over matrix see you there

Reply:
think about this one check matrix